In [1]:
import pandas as pd
import matplotlib.pyplot as plt
plt.rc('font', family='NanumBarunGothic')
import seaborn as sns
from google.colab import drive
drive.mount("/gdrive")

Mounted at /gdrive


In [2]:
work_df = pd.read_csv("/gdrive/My Drive/상권배후지-직장인구.csv",encoding='cp949')
work_df.head()

기준_년_코드  기준_분기_코드 상권_구분_코드 상권_구분_코드_명    상권_코드          상권_코드_명  총_직장_인구_수  \
0     2022         3        A       골목상권  2111090          강일동주민센터        744   
1     2022         3        A       골목상권  2111089     상일여고(상일초등학교)       1080   
2     2022         3        A       골목상권  2111088            고덕동성당       1187   
3     2022         3        A       골목상권  2111087  고덕중학교(고덕2동주민센터)        743   
4     2022         3        A       골목상권  2111086             한영중고        862   

   남성_직장_인구_수  여성_직장_인구_수  연령대_10_직장_인구_수  ...  남성연령대_30_직장_인구_수  \
0         547         197               0  ...               111   
1         716         364               0  ...               146   
2         964         223               0  ...                55   
3         270         473               0  ...                54   
4         441         421               1  ...                75   

   남성연령대_40_직장_인구_수  남성연령대_50_직장_인구_수  남성연령대_60_이상_직장_인구_수  여성연령대_10_직장_인구_수  \
0               143               143                   95                 0   
1               184               192                  136                 0   
2               178               388                  333                 0   
3                69                77                   53                 0   
4                97               157                   79                 0   

   여성연령대_20_직장_인구_수  여성연령대_30_직장_인구_수  여성연령대_40_직장_인구_수  여성연령대_50_직장_인구_수  \
0                35                50                52                47   
1                65                70                76               105   
2                30                37                52                76   
3                68               103               115               131   
4                41               121               112               127   

   여성연령대_60_이상_직장_인구_수  
0                   13  
1                   48  
2                   28  
3                   56  
4                   20  

[5 rows x 27 columns]

In [3]:
work_df1 = work_df[(work_df['기준_년_코드']==2022)&(work_df['기준_분기_코드']==3)]

In [4]:
work_df1

기준_년_코드  기준_분기_코드 상권_구분_코드 상권_구분_코드_명    상권_코드          상권_코드_명  \
0        2022         3        A       골목상권  2111090          강일동주민센터   
1        2022         3        A       골목상권  2111089     상일여고(상일초등학교)   
2        2022         3        A       골목상권  2111088            고덕동성당   
3        2022         3        A       골목상권  2111087  고덕중학교(고덕2동주민센터)   
4        2022         3        A       골목상권  2111086             한영중고   
...       ...       ...      ...        ...      ...              ...   
1085     2022         3        A       골목상권  2110005              세검정   
1086     2022         3        A       골목상권  2110004           대신고등학교   
1087     2022         3        A       골목상권  2110003          세검정초등학교   
1088     2022         3        A       골목상권  2110002          독립문역 1번   
1089     2022         3        A       골목상권  2110001           이북5도청사   

      총_직장_인구_수  남성_직장_인구_수  여성_직장_인구_수  연령대_10_직장_인구_수  ...  \
0           744         547         197               0  ...   
1          1080         716         364               0  ...   
2          1187         964         223               0  ...   
3           743         270         473               0  ...   
4           862         441         421               1  ...   
...         ...         ...         ...             ...  ...   
1085       2723        1465        1258               9  ...   
1086        470         276         194               6  ...   
1087       4837        2723        2114              16  ...   
1088        689         209         480               7  ...   
1089       5968        4230        1738              11  ...   

      남성연령대_30_직장_인구_수  남성연령대_40_직장_인구_수  남성연령대_50_직장_인구_수  \
0                  111               143               143   
1                  146               184               192   
2                   55               178               388   
3                   54                69                77   
4                   75                97               157   
...                ...               ...               ...   
1085               361               399               341   
1086                63                61                59   
1087               684               764               595   
1088                46                59                37   
1089               935              1087              1267   

      남성연령대_60_이상_직장_인구_수  여성연령대_10_직장_인구_수  여성연령대_20_직장_인구_수  \
0                      95                 0                35   
1                     136                 0                65   
2                     333                 0                30   
3                      53                 0                68   
4                      79                 0                41   
...                   ...               ...               ...   
1085                  211                 6               197   
1086                   55                 1                29   
1087                  434                 9               379   
1088                   24                 1               136   
1089                  368                 4               518   

      여성연령대_30_직장_인구_수  여성연령대_40_직장_인구_수  여성연령대_50_직장_인구_수  \
0                   50                52                47   
1                   70                76               105   
2                   37                52                76   
3                  103               115               131   
4                  121               112               127   
...                ...               ...               ...   
1085               354               321               282   
1086                46                51                54   
1087               627               458               422   
1088               157               107                65   
1089               500               296               259   

      여성연령대_60_이상_직장_인구_수  
0                      13  
1                      48  
2    

In [8]:
work_20 = work_df1['연령대_20_직장_인구_수']
work_30 = work_df1['연령대_30_직장_인구_수']
work_40 = work_df1['연령대_40_직장_인구_수']
work_20_sum = work_20.sum()
work_30_sum = work_30.sum()
work_40_sum = work_40.sum()
print('서울시 평균 20대 직장인구 수: ',work_20_sum/1090,'명')
print('서울시 평균 30대 직장인구 수: ',work_30_sum/1090,'명')
print('서울시 평균 40대 직장인구 수: ',work_40_sum/1090,'명')

서울시 평균 20대 직장인구 수:  985.1183486238532 명
서울시 평균 30대 직장인구 수:  1539.4073394495413 명
서울시 평균 40대 직장인구 수:  1533.8834862385322 명


30대 직장인구 수와 40대 직장인구 수가 비슷해보인다. 통계적으로 이들의 평균이 같은지 알아보자.

In [9]:
work_30_list = work_30.tolist()
work_40_list = work_40.tolist()

In [10]:
import scipy.stats

scipy.stats.ttest_ind(work_30_list, work_40_list, equal_var=False)

Ttest_indResult(statistic=0.05408034067515478, pvalue=0.9568761189850742)

p=value의 값이 약 0.957 정도로 높은 값이 나왔다. 따라서 30대 직장인구 수와 40대 직장인구 수는 통계적으로 평균 차이가 없다.